testing/testing.go

```diff
@@ -13,6 +13,8 @@ import (
 
 	"github.com/status-im/status-go/geth/common"
 	"github.com/status-im/status-go/geth/params"
+
+	_ "github.com/stretchr/testify/suite" // required to register testify flags
```

Comment:
@dshulyak If any test file uses `testify/suite`, testify flags will be registered. Why do we need to provide these flags to any file that imports this package?

Reply:
it needs to be registered before this line will be executed https://github.com/status-im/status-go/blob/develop/testing/testing.go#L48

Reply:
Ok, I managed to reproduce this issue. We really should get rid of `init()` and all global variables in this package... I think it's ok to have this as a quick and temporary solution.